From 6bc041097a1d27a33d15ba2823c4b8a33130e2a0 Mon Sep 17 00:00:00 2001
From: Michael Zimmermann <sigmaepsilon92@gmail.com>
Date: Thu, 7 Nov 2024 23:16:59 +0100
Subject: [PATCH 2/2] Fix build on alpine Linux

---
 common.c | 1 +
 config.c | 1 +
 dhcp6c.c | 1 +
 3 files changed, 3 insertions(+)

diff --git a/common.c b/common.c
index 6c5935f..c1bfc00 100644
--- a/common.c
+++ b/common.c
@@ -75,6 +75,7 @@
 #include <stdarg.h>
 #include <syslog.h>
 #include <stdlib.h>
+#include <bsd/stdlib.h>
 #include <unistd.h>
 #include <string.h>
 #include <err.h>
diff --git a/config.c b/config.c
index d83446f..546fc38 100644
--- a/config.c
+++ b/config.c
@@ -44,6 +44,7 @@
 
 #include <syslog.h>
 #include <stdlib.h>
+#include <bsd/stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <ifaddrs.h>
diff --git a/dhcp6c.c b/dhcp6c.c
index 853ab93..af75c3b 100644
--- a/dhcp6c.c
+++ b/dhcp6c.c
@@ -64,6 +64,7 @@
 #include <stdarg.h>
 #include <syslog.h>
 #include <stdlib.h>
+#include <bsd/stdlib.h>
 #include <unistd.h>
 #include <string.h>
 #include <err.h>
-- 
2.47.0

